knowledge-base/actions/elgohr/publish-docker-github-action/action-security.yml

```diff
@@ -0,0 +1,2 @@
+name: 'Publish Docker'
+#GITHUB_TOKEN not used
```

Comment:
I think the permission here would be `packages: write`, `to publish image to ghcr` view [here](https://github.com/elgohr/Publish-Docker-Github-Action/tree/33a481be3e179353cb7793a92b57cf9a6c985860#registry)